src/script_opt/CPP/Inits.cc

```diff
@@ -548,7 +564,15 @@ void CPPCompile::GenLoad()
 
 	Emit("register_scripts__CPP(%s, standalone_init__CPP);", Fmt(total_hash));
 
-	// Spit out the placeholder script.
+	// Spit out the placeholder script, and any associated module
+	// definitions.
+	for ( auto& m : module_names )
```

Comment:
Prefer `const auto&` whenever it doesn't need to be modified.

Reply:
Ditto.